Src/Extern/openPMD-api/AMReX_ParticlesOPENPMD.H

```diff
@@ -0,0 +1,79 @@
+#ifndef AMREX_PTL_OPENPMD_API_H
+#define AMREX_PTL_OPENPMD_API_H
+
+/**
+*   This file is included from AMReX_ParticleCounter.H
```

Comment:
Otherwise this will be assigned to `struct AMReX_PtlCounter` below:
```suggestion
/** @file This file is included from AMReX_ParticleCounter.H
```